PulseSampleApp/app/src/main/java/com/ooyala/sample/players/AbstractHookActivity.java

```diff
@@ -61,8 +62,8 @@ public void invokeDefaultOnBackPressed() {
 
   /** Start Activity methods for Skin **/
   @Override
-  protected void onStop() {
-    super.onStop();
+  protected void onPause() {
+    super.onPause();
```

Comment:
You rename method, but log is old: 
Log.d(TAG, "Player Activity Stopped");
could you change this log or delete if it is not used.

Reply:
done.